cmd/tools/cassandra/main.go

```diff
@@ -27,5 +27,5 @@ import (
 )
 
 func main() {
-	cassandra.RunTool(os.Args)
+	cassandra.RunTool(os.Args) //nolint:errcheck
```

Comment:
I think explicit add nolint comment might be better because:
1. it will avoid people guessing why you do `_ = cassandra.RunTool()`
2. it might save tiny assignment operation

Will keep it that way